service: fix logger behavior to show apiClassName

summerbabybiu opened the pull request:
When print error message, logger should print `apiClassName` instead of
`apiName`.

Fixes: https://github.com/akyuujs/akyuu/issues/23

##### Checklist

- [x] commit message follows Node.js [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

##### Affected core submodule(s)

service

XadillaX reviewed (changes requested):
Some nits:

1. plz follow the hint in input textarea;
2. The first line of commit message should not longer than 50;
3. It seems that your commit user information is not correct.
4. `fix` should be `Fixes`.

XadillaX commented:
You may follow Node.js [commit message guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines).

summerbabybiu committed:
service: fix logger behavior to show apiClassName

When print error message, logger should print `apiClassName` instead of
`apiName`.

Fixes: https://github.com/akyuujs/akyuu/issues/23

summerbabybiu commented:
@XadillaX 应该改好了~~

XadillaX commented:
@summerbabybiu 

## Commit Message

```
service: fix logger behavior to show apiClassName

When print error message, logger should print `apiClassName` instead of
`apiName`.

Fixes: https://github.com/akyuujs/akyuu/issues/23
```

> First line is title, and it should be shorter than 50. Each line of summary block should be shorter than 72.
>
> Those rules can be found in [Node.js commit message guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines).

## PR information

### Title

service: fix logger behavior to show apiClassName

### Content

```markdown
When print error message, logger should print `apiClassName` instead of
`apiName`.

Fixes: https://github.com/akyuujs/akyuu/issues/23

##### Checklist

- [x] commit message follows Node.js [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

##### Affected core submodule(s)

service
```

#### Preview

<details>
When print error message, logger should print `apiClassName` instead of
`apiName`.

Fixes: https://github.com/akyuujs/akyuu/issues/23

##### Checklist

- [x] commit message follows Node.js [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

##### Affected core submodule(s)

service
</details>

summerbabybiu commented:
@XadillaX  em.....May be ok...

XadillaX commented:
Plz modify the PR information.

summerbabybiu commented:
改好了~ @XadillaX

XadillaX reviewed (approved):
LGTM.

XadillaX commented:
Landed in https://github.com/akyuujs/akyuu/commit/b0deac8e8b2350723b3fbf433f15a6fb8d09333a.
